spec/atom-minimap-linter-spec.js

```diff
@@ -0,0 +1,53 @@
+'use babel';
+
+import { join } from 'path';
+
+const testPath = join(__dirname, 'fixtures', 'test.js');
+const fakeLinter = join(__dirname, 'fake-linter');
+
+describe('MinimapLinterBinding', () => {
+  let workspaceElement;
+  let editorView;
+  let editor;
+  let minimap;
+
+  beforeEach(() => {
```

Comment:
`import { beforeEach } from 'jasmine-fix'`

and then you'll be able to do `beforeEach(async function(){`